docs/content.md

```diff
@@ -115,6 +116,16 @@ nested:
 
 Knowing which frontmatter values are available for a page is not always obvious, but the best method is to look at a similar page and copy the frontmatter from there (or look at the page examples in this document).
 
+### Breadcrumbs
+
+Breadcrumbs are included by default on most layouts. You can chose to optionally disable breadcrumbs by setting the following in the front matter:
```

Comment:
@spencerldixon You say `most layouts` here - is it worth calling out which ones don't? Eg sign-up form pages